content/applications/services/project/tasks/sub-tasks.rst

```diff
@@ -0,0 +1,48 @@
+=========
+Sub-tasks
+=========
+
+When handling project tasks, you can often benefit from splitting the workload into smaller
+**sub-tasks**, making it easier to track progress and manage the work.
+
+Odoo’s sub-task feature is particularly useful in some of those cases:
+
+ - The task workload is too big to handle in one record.
+ - There’s a clear sequence of steps to follow.
+ - Some parts of the workload fall under the scope of different projects and/or assignees.
+
+The original task to which sub-tasks are linked is referred to as the **parent task**. The sub-tasks
+of the parent task are known as **child tasks**.
+
+Creating sub-tasks
+==================
+
+Open the Project app, then go into your desired project and click the task where you want to add a
+sub-task to open it.
+
+Click the :guilabel:`sub-tasks` tab. To add a new task, click :guilabel:`Add a line`, then fill in
+the :guilabel:`Title`. Then, click the :icon:`fa-cloud-upload` (:guilabel:`cloud`) icon to save the
+task manually.
+
+.. tip::
+   You can edit several fields of a sub-task directly from the parent task’s sub-task tab, without
+   opening the individual sub-task. By default, those are: :guilabel:`Title`, :guilabel:`Priority`,
+   :guilabel:`Status`, :guilabel:`Assignees`. Use the :icon:`fa-sliders` (**sliders**) icon to add
+   or remove other fields.
+
+Once the new sub-task is created, the following changes occur in the parent task:
+
+ - The :guilabel:`Allocated time` displays the amount of time allocated to the sub-tasks.
```

Comment:
I asked the PO, and I'll update you once I know their feedback. I noticed that the behavior in master is already a bit different from 17.0, so hopefully this will change soon (if this is unexpected).

Reply:
@auva-odoo Here's the update after getting feedback fom Xavier (xbo). Let me know if it makes sense to you (I'm still on the fence, so it's a genuine question! :))

 - The :guilabel:`Allocated time` displays the amount of time allocated to the parent task, as well
   as time allocated to the sub-tasks. This can be used to ensure that the time allocated to
   sub-tasks does not exceed time allocated to the parent task.
 - The :guilabel:`Timesheets` tab includes a breakdown of time spent on the parent task as well as
   sub-tasks.